lib/redis.js

```diff
@@ -91,8 +91,9 @@ module.exports = CoreObject.extend({
     });
   },
 
-  activeRevision: function(keyPrefix) {
-    var currentKey = keyPrefix + ':current';
+  activeRevision: function(keyPrefix, activationSuffix) {
+    activationSuffix = activationSuffix || 'current';
```

Comment:
I tried to remove this one but quite some tests are still relaying on it and I couldn't figure out how to fix it...